src/components/BounceRulesContainer/Modals/CreateRuleModal/index.jsx

```diff
@@ -125,7 +126,7 @@ const CreateRuleModal = ({
     description,
     enhanced_code: enhancedCode,
     regex,
-  } = newRule;
+  } = { ...newRule };
```

Comment:
Hmm we're spreading an object into another blank object? I think we'll be all good without the spreading 😄